client/components/site-thumbnail.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+export default React.createClass({
+  propTypes: {
+    url: React.PropTypes.string
+  },
+  render () {
+    let block = {
+      position: 'absolute',
+      display: 'block',
+      height: '380',
+      width: '100%',
+      background: 'transparent'
+    }
+    return (
+      <div onClick={ this.stopPropergation }>
```

Comment:
Typo here https://developer.mozilla.org/en/docs/Web/API/Event/stopPropagation

Also, will this work? I can't see a `stopPropergation` function on this component to call.